### PR TITLE
Create CODEOWNERS

### LINKED ISSUE #210: Create CODEOWNERS File
**Is your feature request related to a problem? Please describe.**
We need a CODEOWNERS file.

**Describe the solution you'd like**
https://help.github.com/en/articles/about-code-owners

**Describe alternatives you've considered**
Assigning issues manually is way to tedious.

### PR DESCRIPTION
**Pull request by LanikSJ**
## Description
Create CODEOWNERS File

## Related Issue
This will allow us to know who can approve code etc.
@nlowe @BabylonZeus @krazyito65 @Zweih if you have a problem with this please let me know.

## Motivation and Context
Fixes #210 

## How Has This Been Tested?
Travis CI

## Types of changes

-   [ ] Bug fix (non-breaking change which fixes an issue)
-   [x] New feature (non-breaking change which adds functionality)
-   [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

-   [x] My code follows the code style of this project.
-   [x] My change requires a change to the documentation.
-   [x] I have updated the documentation accordingly.
-   [x] I have read the [**CONTRIBUTING**](CONTRIBUTING.md) document.

### CODE REVIEW

**Comment by nlowe** [comment]
@LanikSJ What's the expectations for being on this list?

**Comment by LanikSJ** [comment]
@nlowe I expect absolutely nothing.  I do this on my own free time as well as many others I imagine.  If you're not comfortable being on this list I can remove your name.  When someone creates a PR it will show your name next to the file they're editing / contributing.  By chance it's possible someone will assign the PR to you.  The chance would be remote since I'm the repo owner but it's still there.

**Comment by nlowe** [comment]
@LanikSJ Makes sense, just wanted some background. I'm happy to help out on issues / reviews for PowerShell where I can.

**Comment by LanikSJ** [comment]
Most excellent. 😁 

That's about all I was hoping for that folks would be willing to volunteer their time.  As this project has grown outside of my OS / Slack version, I'm hoping the community at large would be willing to help me fill in the blanks etc.

Again I appreciate everyones contribution.  And there's absolutely **NO** expectation from anyone here.

File: .github/CODEOWNERS [hunk 1]
```diff
@@ -0,0 +1,7 @@
+*                             @LanikSJ
+README.md                     @krazyito65 @Zweih
+dark-theme.css                @krazyito65 @Zweih
+event-listener.js             @krazyito65 @Zweih
+slack-dark-mode.ps1           @niowe @BabylonZeus @krazyito65
```
Comment (nlowe) [inline .github/CODEOWNERS]: Pretty sure this is supposed to be `@nlowe` not `@niowe`

Comment (LanikSJ) [reply .github/CODEOWNERS]: 😭

**Comment by LanikSJ** [comment]
@nlowe you're catching on. 😁

That was my entire reason for this. So someone else can approve my PRs.

**Comment by LanikSJ** [comment]
Looks like the majority have it.

**Pull request merged**